parcels/field.py

```diff
@@ -626,6 +626,8 @@ def interpolator2D(self, ti, z, y, x):
             return val
         elif self.interp_method is 'cgrid_tracer':
             return self.data[ti, yi+1, xi+1]
+        elif self.interp_method is 'cgrid_velocity':
+            raise RuntimeError(self.interp_method+" must be interpolated as VectorField")
```

Comment:
Also add name of Field to error message? And be more explicit by adding `(e.g. fieldset.UV[])`?